### PR TITLE
feat(ImageBlock): add BlankState

### PR DESCRIPTION
**Pull request by celine-s**
Tickets: [Blank State](https://app.clickup.com/t/867767mr9), [Loading state](https://app.clickup.com/t/8677a1z3n)

### CODE REVIEW

File: packages/shared/src/components/AddImagesButton.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        const XInsideComponent = e.clientX - left;
+        const YInsideComponent = e.clientY - top;
+        setMenuPosition([XInsideComponent, YInsideComponent]);
+    };
+
+    return (
+        <button
+            ref={buttonRef}
+            className={joinClassNames([
+                'tw-font-body tw-relative tw-text-[14px] tw-text-text-weak tw-border tw-rounded-[4px] tw-flex tw-items-center tw-justify-center tw-cursor-pointer tw-gap-3 tw-w-full',
```
[the remark this answers is not in the document]
Comment (celine-s) [reply packages/shared/src/components/AddImagesButton.tsx:59]: oh true. Nice - thanks!!

File: packages/shared/src/components/AddImagesButton.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    isLoading,
+    fillParentContainer,
+    onAssetChooseClick,
+    onUploadClick,
+}: AddImagesButtonProps) => {
+    const [isDraggingOver, setIsDraggingOver] = useState(false);
+    const [menuPosition, setMenuPosition] = useState<[number, number] | undefined>();
+    const buttonRef = useRef<HTMLButtonElement>(null);
+
+    const handleDrop: React.DragEventHandler<HTMLButtonElement> = (e) => {
```
[the remark this answers is not in the document]
Comment (celine-s) [reply packages/shared/src/components/AddImagesButton.tsx:39]: updated

File: packages/shared/src/components/AddImagesButton.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    label: string;
+    secondaryLabel?: string;
+    icon?: JSX.Element;
+    onDrop: (files: FileList) => void;
+    fillParentContainer?: boolean;
+    onUploadClick: () => void;
+    onAssetChooseClick: () => void;
+};
+
+export const AddImagesButton = ({
```
[the remark this answers is not in the document]
Comment (celine-s) [reply packages/shared/src/components/AddImagesButton.tsx:25]: renamed 👍

**Review by fulopdaniel — APPROVED** [review]

File: packages/image-block/src/ImageCaption.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import { hasRichTextValue, joinClassNames } from '@frontify/guideline-blocks-shared';
+
+type ImageCaptionProps = {
+    name?: string;
+    isEditing: boolean;
+    description?: string;
+    onNameChange: (value: string) => void;
+    onDescriptionChange: (value: string) => void;
+};
+export const ImageCaption = ({
```
Comment (Walkeezy) [inline packages/image-block/src/ImageCaption.tsx:13]: Please use guideline design tokens for image title & image caption.

Comment (celine-s) [reply packages/image-block/src/ImageCaption.tsx:13]: added the design tokens. To use the correct style I think its best to do it directly in this ticket: https://app.clickup.com/t/8677a21bv. Wdyt?

Comment (Walkeezy) [reply packages/image-block/src/ImageCaption.tsx:14]: Good for me 👍🏻

**Review by Walkeezy — APPROVED** [review]

**Pull request merged**